experimental/bies/README.md

```diff
@@ -20,11 +19,11 @@ For example, suppose you had the following BIES matrix:
 | 0.01  | 0.01  | 0.01  | 0.97  |
 | 0.97  | 0.01  | 0.01  | 0.01  |
 | 0.01  | 0.01  | 0.97  | 0.01  |
-```
+```rust
```

Comment:
Why does this say `rust`?  Seems to cause formatting issues.

Reply:
Not sure, that comes from cargo-readme. Looks like I can work around it by using "not-actually-code" on both sides.